core/templates/dev/head/pages/story_editor/main_editor/StoryNodeEditorDirective.js

```diff
@@ -82,7 +82,13 @@ oppia.directive('storyNodeEditor', [
           $scope.getSkillEditorUrl = function(skillId) {
             return '/skill_editor/' + skillId;
           };
-
+          // Regex pattern for exploration id.
+          $scope.explorationIdPattern = /^[a-zA-Z0-9_-]*$/;
```

Comment:
Maybe add a comment saying why you wrote `*` instead of `+` above this. Basically, why `EXPLORATION_AND_SKILL_ID_PATTERN` is not used here.

Reply:
Done